enzyme/Enzyme/EnzymeLogic.cpp

```diff
@@ -2611,15 +2611,16 @@ Function *EnzymeLogic::CreatePrimalAndGradient(
   assert(!todiff->empty());
 
   ReturnType retVal =
-      returnValue ? (dretPtr ? ReturnType::ArgsWithTwoReturns
-                             : ReturnType::ArgsWithReturn)
-                  : (dretPtr ? ReturnType::ArgsWithReturn : ReturnType::Args);
+      fwdMode       ? ReturnType::Return
```

Comment:
Technically won't foward mode need to be either return or two returns, depending on whether the return is active?

Reply:
Yes. I wanted to deal with that later.